appveyor.yml

```diff
@@ -7,13 +7,13 @@ init:
 environment:
   matrix:
     - nodejs_version: '8'
-      webpack_version: latest
+      webpack_version: '4.0.0-beta.0'
```

Comment:
Please don't alter this, we have a boilerplate package (`webpack-defaults`) which automatically updates these files

Reply:
What is not working here atm ?

Reply:
Sorry about that 😅 I'll revert that change.